cvat-ui/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cvat-ui",
-  "version": "1.0.1",
+  "version": "1.1.2",
```

Comment:
Hmm.. Why 1.1.2? Right now in develop I see 1.0.1
Probably it should be version 1.0.2